README.md

```diff
@@ -18,7 +18,7 @@ them](https://www.facebook.com/business/help/606443329504150?helpref=faq_content
 1. Clone and change to the directory:
 
     ```sh
-    git clone git@github.com:mozilla/fx-private-relay.git
+    git clone github.com/mozilla/fx-private-relay.git
```

Comment:
```sh
git clone github.com/mozilla/fx-private-relay.git
fatal: repository 'github.com/mozilla/fx-private-relay.git' does not exist
```

I think this is failing because this isn't an SSH uri or HTTPS uri.

These are the clone values I get from the GitHub clone UI:
- SSH: `git@github.com:mozilla/fx-private-relay.git`
- HTTPS: `https://github.com/mozilla/fx-private-relay.git`

Reply:
yes, sorry my bad